temporal-sdk/src/main/java/io/temporal/client/WorkflowStubImpl.java

```diff
@@ -131,45 +131,71 @@ public WorkflowExecution start(Object... args) {
   }
 
   @Override
-  public <R> WorkflowUpdateHandle<R> updateWithStart(
-      UpdateWithStartWorkflowOperation<R> updateOperation, Object... startArgs) {
+  public <R> WorkflowUpdateHandle<R> startUpdateWithStart(
+      @Nonnull UpdateOptions<R> updateOptions,
+      Object[] updateArgs,
+      @Nonnull WithStartWorkflowOperation<?> startOp) {
     if (options == null) {
-      throw new IllegalStateException("Required parameter WorkflowOptions is missing");
+      throw new IllegalStateException(
+          "Required parameter WorkflowOptions is missing in WorkflowStub");
     }
-    if (!updateOperation.markInvoked()) {
-      throw new IllegalStateException("UpdateWithStartWorkflowOperation was already executed");
+    if (options.getWorkflowIdConflictPolicy() == null) {
+      throw new IllegalStateException(
+          "Required parameter WorkflowIdConflictPolicy in WorkflowOptions is missing in WorkflowStub");
```

Comment:
Important change: we're making the `WorkflowIdConflictPolicy` a required field for UwS across all SDKs.

Reply:
Yeah +1 to this wording